packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
 
 To learn more about Synthetics capabilities, check out the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html).
 
+
+### Canary Schedule
+You can set the canary `schedule` property by assigning it a value with the `synthetics.Schedule` [class](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-synthetics.Schedule.html).
```

Comment:
"set the canary schedule property" is not really a helpful statement, how about something like "Specify the schedule on which a canary runs by providing a `Schedule` value to the `schedule` property."?